README.md

```diff
@@ -1,12 +1,65 @@
-# TypeScript Lab 1
+# TypeScript Lab 1: Basic Typing
 
-## Instructions
+## Learning Goals
 
-Write a TypeScript function that adds a number of spaces to the front of a
-`string`. The signature of the method should be:
+- Write a function with TypeScript 
+- Use TypeScript type annotation 
 
-```typescript
-function prepend(numSpaces: number, inputString: string) {
-  // implementation here
-}
+## Introductions to Labs 
+
+Welcome to your first lab in this course! Throughout this course, you will work 
+on various assignments as you learn new concepts. 
+
+All assignments are hosted on GitHub. In order to work on them, however, you 
+will need to complete work on your local machine. The general process is:
+
+- Click the blue "Fork" button in Canvas (found in the top right corner of the
+  assignment)
+- Create a personal copy (a 'fork') of the assignment in GitHub
+- Download your personal copy (referred to as 'cloning') to your computer
+- Complete the required work
+- Submit your completed work to Canvas
+
+This process should feel familiar from the prework. With that refresher, let's
+get started on your first TypeScript lab. 
+
+## The Activity 
+
+Write a TypeScript function called `prepend` that takes in two parameters, a 
+number and a string. The function should add the provided number amount of 
+spaces in front of the provided string and console log the result. 
+
```

Comment:
How about: "The function should add the number of spaces indicated by the first parameter in front..."